# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/ClientLayerManager.h
# Commit: 641765261f49
# Full Hash: 641765261f499a1a4712c0c02a14b58b69ecabaa
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2019-07-26 21:36:38
# Regressor Bug: 1569110
# File Overlap Count: 1
# Description:
#   Bug 1569110 - Add ClientLayerManager::Initialize() r=nical
#   
#   Some initialization handlings of ClientLayerManager exists in nsBaseWidget::CreateCompositor(). It is not good. Move them to ClientLayerManager::Initialize().
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D39476
# ==============================================================================

diff -r a6fa09658817 -r 641765261f49 gfx/layers/client/ClientLayerManager.h
--- a/gfx/layers/client/ClientLayerManager.h	Fri Jul 26 13:25:35 2019 +0000
+++ b/gfx/layers/client/ClientLayerManager.h	Fri Jul 26 09:31:57 2019 +0000
@@ -53,7 +53,8 @@
 
  public:
   explicit ClientLayerManager(nsIWidget* aWidget);
-
+  bool Initialize(PCompositorBridgeChild* aCBChild, bool aShouldAccelerate,
+                  TextureFactoryIdentifier* aTextureFactoryIdentifier);
   void Destroy() override;
 
  protected: